spacemacs/packages.el

```diff
@@ -1393,6 +1393,9 @@ ignored)."
         "sl"  'helm-semantic-or-imenu
         "hb"  'helm-bookmarks
         "hl"  'helm-resume
+        "hi"  'helm-info-at-point
+        "hm"  'helm-man-woman
+        "hL"  'helm-locate-library
```

Comment:
If it is related to elisp then it should go behind `SPC m h L` with the other key bindings in `spacemacs/keybindings.el`